iceoryx_posh/test/integrationtests/test_mq_interface_startup_race.cpp

```diff
@@ -33,12 +34,12 @@ using iox::runtime::MqMessage;
 using iox::runtime::MqMessageType;
 using iox::runtime::MqRuntimeInterface;
 
-using MQueue = iox::posix::MessageQueue;
-
-constexpr char MqRouDiName[] = "/roudi";
-constexpr char MqAppName[] = "/racer";
 
+#if !defined(__APPLE__)
```

Comment:
Yes, otherwise I get an unused variable warning and then I have to add clang warning suppression pragmas to avoid that you call my firstborn Jean Luc Runkelruebe